concourse/scripts/import_bosh_ca.sh

```diff
@@ -3,4 +3,8 @@ set -eu
 
 echo "Adding bosh-CA to root certificates"
 tar -xzf bosh-CA/bosh-CA.tar.gz -C /usr/local/share/ca-certificates bosh-CA.crt
-update-ca-certificates
+
+UPDATE_OUTPUT=$(update-ca-certificates 2>&1)
+set +e
```

Comment:
I'm assuming we're doing this because `update-ca-certificates` can print nothing to `std{out,err}` during normal operation?

Reply:
We're doing this because we can't run it as one command with grep as we need to ignore the grep exit code

Reply:
(Also update-ca-certificates loggs to stderr)

Reply:
I'm saying that `grep` returns `0` if at least one line is printed. If `update-ca-certificates` always prints something normally (which I think it does) then we can just one-line the whole thing?

```
$ cat /etc/lsb-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=14.04
DISTRIB_CODENAME=trusty
DISTRIB_DESCRIPTION="Ubuntu 14.04.5 LTS"
$ update-ca-certificates | grep .
Updating certificates in /etc/ssl/certs... 0 added, 0 removed; done.
Running hooks in /etc/ca-certificates/update.d....done.
root@4b342991654d:/# echo $?
0
root@4b342991654d:/# update-ca-certificates | grep z
root@4b342991654d:/# echo $?
1
```

Reply:
Currently it prints one warning, which could go away in the future. In that case we don't want the script to fail.

Reply:
Ok, I see what you mean, interestingly it doesn't say these things on bosh when the warning is written.

Reply:
So we run this command on Alpine and the command doesn't output anything else there. As we should be prepared that this warning might go away in the future we should keep the commands separate.